src/styles/card.css

```
@@ -0,0 +1,91 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: var(--white-color);
+}
+
+.card__photo {
+  margin: 0 auto 40px;
+  
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--black-color);
+}
+
+.card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--gray-color);
+}
+
+.card__reviews {
+  margin-bottom: 24px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__reviews-amount {
+  font-size: 10px;
+  color: var(--black-color);
+}
+
+.card__price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price-title {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--gray-color);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--black-color);
+}
+
+.card__button {
+  display: block;
+  height: 40px;
+  
+  background: var(--blue-color);
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  color: var(--white-color);
+  text-align: center;
+  line-height: 40px;
+
+  transition: 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```

---

Fixed, thank you!